README.md

```diff
@@ -0,0 +1,99 @@
+# Furnace
```

Comment:
What's wrong with .rst? :C
Also, please remove that, if you're going with .md

Reply:
In fact, you cannot simply remove the .rst, since that's needed for pypi. Also, this README.md is outdated compared to .rst, lots of things are wrong in it.

Reply:
Thx.